movieapp/src/containers/AddEditMovieForm/AddEditMovieForm.tsx

```diff
@@ -0,0 +1,79 @@
+import {Button} from "../../components/Button/Button";
+import {useTranslation} from "react-i18next";
+
+type MovieProps = {
+    title: string | '';
+    description: string | '';
+    key: string | '';
+}
+
+type MovieFormProps = {
+    onCloseModal: Function;
+    onAddMovie: Function;
+    currentValue:MovieProps;
+}
+
+export function AddEditMovieForm(props:MovieFormProps): JSX.Element {
+    const {t} = useTranslation();
+    const currentValueToEdit = props.currentValue;
+
+    const submitForm = (e:any) => {
+        const title = e.target.elements.movieTitle.value;
+        const description = e.target.elements.movieDescription.value;
+        let key = e.target.elements.movieKey.value;
```

Comment:
avoid usage of `let` use pure functions approach

Reply:
fixed